services/cronjobs.factory.js

```diff
@@ -0,0 +1,5 @@
+const cron = require('node-cron');
+const logger = require('./logger.factory');
+const factory = require('./cronjobs.service')({ cron, logger });
```

Comment:
I wld name that cronJobService , cause u exporting the service and not the factory :)